meta-resin-common/recipes-support/resin-vars/resin-vars/config-json.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Config.json watcher
+Requires=mnt-boot.mount
+After=mnt-boot.mount
+
+[Service]
+Type=oneshot
+ExecStart=/bin/echo 'config.json changed'
+
+[Install]
+WantedBy=mnt-boot.mount
```

Comment:
Won't mnt-boot only be activated when you remount /mnt/boot? At which point the file watcher and this service need to be restarted since the /mnt/boot/config.json they point at will have changed by virtue of the mount (it certainly needs to be activated the first time /mnt/boot is activated or otherwise the downstream services will never be triggered for the initial version of the file)